design/one-pager-ignore-changes.md

```diff
@@ -0,0 +1,354 @@
+
+Some examples of this case:
+- [AutoScalingGroup DesireCapacity], where when using an autoscaler, the
+  parameter gets changed externally.
+- [DynamoDB Tables] with using autoscaling ([Policy] + [Target]). The
+  autoscaler scales the read and write capacity, and then Crossplane resets
+  it back to the `readCapacity` and `writeCapacity` defined on the Table,
+  which effectively makes autoscaling useless.
+- [NodeGroup] while using a node autoscaler. There is a race condition
+  between crossplane and node autoscaler. Node autoscaler increases the
+  `desiredSize` due to resource requirement but crossplane decrease it to
+  `desiredSize` set in composition.
+- GCP [Node Pool], where when `management.autoUpgrade` is enabled. After
+  creation, the provider late initializes `spec.forProvider.version`,
+  which when the nodes get upgraded starts conflicting with the 
+  external nodes version. This causes an endless loop of downgrades
+  and upgrades.
+- Tags are a use case that affects multiple resources. Many cloud users
+  have external systems that add tags to resources for billing, tracking
+  or similar purposes. This can conflict with the tags set in 
+  `spec.forProvider.tags`.
+- There is an use case with [AzureAD Group] where initially the group is
+  created certain with `members` and `owners` but is then managed externally
+  over time as members need to be added or removed. For now the
+  workaround is to use terraform provider with [ignore_changes].
+
+In these cases, it would make sense to make Crossplane ignore select
+parameters during updates. Terraform addresses this use case with its
+[ignore_changes] functionality.
+
+An additional benefit of ignoring some fields would be that it would be
+possible to have different Crossplane Compositions managing a single
+external resource. This could open up some interesting new possibilities
+for Crossplane.
+
+The obvious way would be to just not set the field in the `spec.forProvider`.
+However, due to [Late Initialization], some fields are being set in
+`spec.forProvider` without any user input. Plus there are some cases
+where the field is required/wanted upon creation, but not on updates.
+
+So in this document, the aim will be to introduce a solution in Crossplane
+for ignoring some managed resource parameters during updates.
+
+## Goals
+
+* Design a functionality that allows for ignoring select fields during
+  resource updates.
+
+## Proposal
+
+Proposed solution is to use the new [managementPolicy] field which came with
+the [ObserveOnly] feature and add a new management policy that will 
+skip late initialization. The loss the information that the
+late initialization was providing would be offset by the `status.atProvider`
+which contains the resource state on the provider side and is being
+added alongside [ObserveOnly].
+
+In addition to the new management policy, we will also add a new field
+`spec.initProviders` which would contain parameters that should only be
```

Comment:
```suggestion
`spec.initProvider` which would contain parameters that should only be
```

Nit. This appears a few places in the document. I think it should always be `spec.initProvider` (singular).